ddev/src/ddev/e2e/agent/docker.py

```diff
@@ -16,6 +17,8 @@
 
     from ddev.utils.fs import Path
 
+AGENT_VERSION_REGEX = r'^datadog/agent:[67](?:$|\.(\d\d\.\d(?:$|-jmx$)|$))'
```

Comment:
Will we remember to bump the `[67]` part when agent 8 comes out? :)

Reply:
I would `\d+` 😅

Reply:
Same. Don't see the benefit of restricting this number, but do see the cost :)

Reply:
Sure. I don't feel strongly about it either way.